src/universal/components/CreateCard/CreateCard.js

```diff
@@ -80,18 +69,41 @@ CreateCard.propTypes = {
   styles: PropTypes.object
 };
 
-const labelStyles = {
-  display: 'inline-block',
+const borderRadius = '.1875rem';
```

Comment:
alrighty, i'm cool with whatever you decide, just for fun though, here's what happens in chrome when you set font to extra large: http://recordit.co/pTBinNyjey